tools/redis.md

```diff
@@ -0,0 +1,34 @@
+---
+permalink: /redis
+layout: post
+title: Redis
+link: https://docs.redislabs.com/latest/rs/administering/product-lifecycle/#end-of-life-schedule
```

Comment:
This link is for Redis Enterprise, are you sure this applies for the open source Redis as well?

Reply:
I also double check this earlier. The open source redis [https://redis.io/support](https://redis.io/support) shows the redislab docs. Also seeing the version matches with the one on redis.io I assumed they are equal. :)

Reply:
But I'll just href it to redis.io since the page says "Redis website"

Reply:
Pushed changes for this discussion